storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/queries/DeleteQueryTest.java

```diff
@@ -58,6 +65,77 @@ public void whereArgsShouldNotBeNull() {
         assertThat(deleteQuery.whereArgs().isEmpty()).isTrue();
     }
 
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = NullPointerException.class)
+    public void completeBuilderShouldNotAllowNullUriObject() {
+        //noinspection ConstantConditions
+        DeleteQuery.builder()
+                .uri(mock(Uri.class))
+                .uri((Uri) null)
+                .build();
+    }
+
+    @SuppressWarnings("ConstantConditions")
+    @Test(expected = NullPointerException.class)
+    public void completeBuilderShouldNotAllowNullUriString() {
+        //noinspection ConstantConditions
+        DeleteQuery.builder()
+                .uri(mock(Uri.class))
+                .uri((String) null)
+                .build();
+    }
+
+    @Test(expected = IllegalStateException.class)
+    public void completeBuilderShouldNotAllowEmptyUriString() {
+        DeleteQuery.builder()
+                .uri(mock(Uri.class))
+                .uri("")
+                .build();
+    }
+
+    @Test
+    public void completeBuilderShouldUpdateUriObject() {
+        Uri oldUri = mock(Uri.class);
+        Uri newUri = mock(Uri.class);
+
+        DeleteQuery query = DeleteQuery.builder()
+                .uri(oldUri)
+                .uri(newUri)
+                .build();
+
+        assertThat(query.uri()).isEqualTo(newUri);
+    }
+
+    @Test
+    public void completeBuilderShouldUpdateUriString() {
+        Uri oldUri = Uri.parse("content://1");
+        String newUri = "content://2";
+
+        DeleteQuery query = DeleteQuery.builder()
+                .uri(oldUri)
+                .uri(newUri)
+                .build();
+
+        assertThat(query.uri()).isEqualTo(Uri.parse(newUri));
```

Comment:
We can't check links equality here, because we don't store raw `String` field in `Query`